Library/Formula/openhmd.rb

```diff
@@ -0,0 +1,32 @@
+class Openhmd < Formula
+  homepage "http://openhmd.net"
+  url "http://openhmd.net/releases/openhmd-0.1.0.tar.gz"
+  sha1 "186c747399bd9a509ac8300acbae8823fc4fcc79"
+
+  head do
+    url "https://github.com/OpenHMD/OpenHMD.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "hidapi" => :build
+
+  def install
+    args = ["--prefix", prefix,
+            "--disable-debug",
+            "--disable-silent-rules",
+            "--disable-dependency-tracking",
+            "--bindir=#{prefix}/bin"]
+
+    system "./autogen.sh" if build.head?
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    system "#{prefix}/bin/unittests"
```

Comment:
Fine by me, is the openhmd formula sound as is?

Le 21 févr. 2015 à 17:12, Mike McQuaid <notifications@github.com> a écrit :

> In Library/Formula/openhmd.rb:
> 
> > +  depends_on "hidapi" => :build
> > +
> > +  def install
> > +    args = ["--prefix", prefix,
> > +            "--disable-debug",
> > +            "--disable-silent-rules",
> > +            "--disable-dependency-tracking",
> > +            "--bindir=#{prefix}/bin"]
> > +
> > +    system "./autogen.sh" if build.head?
> > +    system "./configure", *args
> > +    system "make", "install"
> > +  end
> > +
> > +  test do
> > +    system "#{prefix}/bin/unittests"
> I don't think we want this formula to be writing into the Python Cellar so pass on that for now.
> 
> —
> Reply to this email directly or view it on GitHub.
>